pkgs/by-name/gk/gk-cli/package.nix

```diff
@@ -0,0 +1,121 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+  writeShellScript,
+  curl,
+  jq,
+  common-updater-scripts,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gk-cli";
+  version = "2.1.0";
+
+  src = (
+    finalAttrs.passthru.sources.${stdenv.system}
+      or (throw "gk-cli ${finalAttrs.version} unsupported system: ${stdenv.system}")
+  );
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm555 gk -t $out/bin/
+
+    installShellCompletion --bash ./**/gk.bash
+    installShellCompletion --fish ./**/gk.fish
+    installShellCompletion --zsh ./**/_gk
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/gk \
+      --prefix PATH : "${lib.makeBinPath [ coreutils ]}"
+  '';
+
+  passthru = {
+    sources =
+      let
+        base_url = "https://github.com/gitkraken/gk-cli/releases/download/v${finalAttrs.version}/gk_${finalAttrs.version}_";
+      in
+      {
+        armv6l-linux = fetchzip {
+          url = "${base_url}Linux_arm.zip";
+          hash = "sha256-8d3kzccSw9KszZoTjMNIExQOols/sDQprlqVojk1Eec=";
+          stripRoot = false;
+        };
```

Comment:
I asked which type of ARM it is

The answer is ARMv6 https://github.com/gitkraken/gk-cli/issues/61#issuecomment-2273836042

So i added it

Reply:
Thanks for the effort, @pinage404, much appreciated!!

Reply:
Do you know what is preventing the merge of this PR ?

Reply:
Nothing, it has all the visibility that I'm aware of (approvals label and mentioned in discourse), all checks have passed, except for darwin, which usually take quite some time, hence my earlier nixpkgs-review on `aarch64-darwin`.
The only thing that's left is someone with commit rights to this repo coming across this PR 🙂 
Should be merged in the next few days and within a fortnight at most.